### ISSUE TITLE
Reset "User since" when an account is being deleted

### ISSUE THREAD

**Issue by wagnerand** [mozilla/addons-server#2924]
As per https://bugzilla.mozilla.org/show_bug.cgi?id=1271741 the "User since" value is kept if an account is deleted. We'd probably want to _anonymize_ that as well, maybe just set it to _today_?

**Comment by eviljeff** [comment]
There was too much crazy in that bug - can you summarize _why_ we should anonymise "User since" when it's not personally identifiable data?

**Comment by EnTeQuAk** [comment]
if we're going to anonymize this we should maybe just set it to epoch time and be done with it. I can imagine given the crazy in that bug that setting it to `today` adds more weirdness that could be used to somehow do stuff... or not.

**Comment by wagnerand** [comment]
I am having a hard time following that bug myself. I just copied the remaining piece in https://bugzilla.mozilla.org/show_bug.cgi?id=1271741#c8 to here.

**Comment by wagnerand** [comment]
_epoch time_ is fine with me. I don't think `None` would work as the db field is defined as `NOT NULL`.

**Comment by eviljeff** [comment]
In the absence of a coherent 'why' I vote wonfix.

**Comment by EnTeQuAk** [comment]
I don't see that there is an actual bug too. Deleting this data should be easy enough though with updating https://github.com/mozilla/addons-server/blob/master/src/olympia/users/models.py#L399 and writing a fitting test.

**Comment by lavish205** [comment]
I would love to work on this issue, looking forward for some instructions to start with.

**Comment by wagnerand** [comment]
What information is missing from https://github.com/mozilla/addons-server/issues/2924#issuecomment-227208386?
During triage, if it is decided that this is a good first bug, we will add a mentor who can provide more details if needed.

**Comment by wagnerand** [comment]
During goodfirstbugs triage we decided that the current behavior is ok and does not need to be changed.
